client/fire-webclient/src/main/kotlin/uk/ac/ebi/fire/client/api/FireWebClient.kt

```diff
@@ -63,17 +64,12 @@ internal class FireWebClient(
     }
 
     override fun findByMd5(md5: String): List<FireApiFile> =
-        template.getForObject<Array<FireApiFile>>("/objects/md5/$md5").toList()
+        template.getForObjectOrNull<Array<FireApiFile>>("/objects/md5/$md5").orEmpty().toList()
```

Comment:
Here we're actually calling nonexisting endpoints which are giving us the 404. For example, if the md5 is empty we would hit `/objects/md5/` which doesn't exist. Maybe we should not call the endpoints when the values are empty and just return an empty list instead of handling the 404 for a call we know will fail? 🤔

Reply:
error on terminal for me , was md5 not found.

Reply:
Yeah, I thought it was coming from the standard http error but apparently they decided this design but, at least, I think we could save one request since we know for sure empty md5 or path will not return any result don't you think?